zerver/tests/fixtures/kroki.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "name": "Kroki_basic_test_hello_world",
+        "input": "```kroki graphviz\ndigraph G {\nHello->World\n}",
+        "html_output": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"no\"?>\n<!DOCTYPE svg PUBLIC \"-//W3C//DTD SVG 1.1//EN\"\n \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n<!-- Generated by graphviz version 2.44.0 ()\n -->\n<!-- Title: G Pages: 1 -->\n<svg width=\"74pt\" height=\"116pt\"\n viewBox=\"0.00 0.00 74.29 116.00\" xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\">\n<g id=\"graph0\" class=\"graph\" transform=\"scale(1 1) rotate(0) translate(4 112)\">\n<title>G</title>\n<polygon fill=\"white\" stroke=\"transparent\" points=\"-4,4 -4,-112 70.29,-112 70.29,4 -4,4\"/>\n<!-- Hello -->\n<g id=\"node1\" class=\"node\">\n<title>Hello</title>\n<ellipse fill=\"none\" stroke=\"black\" cx=\"33.15\" cy=\"-90\" rx=\"30.59\" ry=\"18\"/>\n<text text-anchor=\"middle\" x=\"33.15\" y=\"-86.3\" font-family=\"Times,serif\" font-size=\"14.00\">Hello</text>\n</g>\n<!-- World -->\n<g id=\"node2\" class=\"node\">\n<title>World</title>\n<ellipse fill=\"none\" stroke=\"black\" cx=\"33.15\" cy=\"-18\" rx=\"33.29\" ry=\"18\"/>\n<text text-anchor=\"middle\" x=\"33.15\" y=\"-14.3\" font-family=\"Times,serif\" font-size=\"14.00\">World</text>\n</g>\n<!-- Hello&#45;&gt;World -->\n<g id=\"edge1\" class=\"edge\">\n<title>Hello&#45;&gt;World</title>\n<path fill=\"none\" stroke=\"black\" d=\"M33.15,-71.7C33.15,-63.98 33.15,-54.71 33.15,-46.11\"/>\n<polygon fill=\"black\" stroke=\"black\" points=\"36.65,-46.1 33.15,-36.1 29.65,-46.1 36.65,-46.1\"/>\n</g>\n</g>\n</svg>"
```

Comment:
This HTML looks like it's intended to be a complete webpage, not something embedded in another page.

Does `kroki` have an option to not do that?  If not, we should open an issue asking for them to do so.

Reply:
I don't understand how this HTML is intended to be a complete webpage. We have the image in SVG format.